javascript/react/fundamentals/what-are-props.md

```diff
@@ -28,28 +28,35 @@ aspects:
 
 ---
 
-# What are `props`?
+# What are props?
 
 ---
 ## Content
 
 As discussed before, `component`s can take input in a similar way to a function, via `props`.
 
-`props` is a plain `JS` object accessible via `this.props`, its properties representing the arguments of a `component`.
+`props` is a plain JS object accessible via `this.props`, its properties representing the arguments of a `component`.
```

Comment:
`component` -> this isn't referencing any actual code so I think it'd be better to write it as _component_